pytential/utils.py

```diff
@@ -61,4 +61,8 @@ def flatten_to_numpy(actx, ary):
     from pytools.obj_array import obj_array_vectorize
     return obj_array_vectorize(actx.to_numpy, result)
 
+
+def sort_arrays_together(*arys, key=str):
+    return zip(*sorted([x for x in zip(*arys)], key=key))
```

Comment:
@inducer, I'm sorting them using `str`. I can change that to `repr`.

Reply:
Ah. I had missed the default arg. IMO that default arg is easy to overlook. Put that into the calling code.